pm/recipient.go

```diff
@@ -271,16 +274,28 @@ func (r *recipient) faceValue(sender ethcommon.Address) (*big.Int, error) {
 		faceValue = new(big.Int).Mul(r.cfg.EV, evMultiplier)
 	}
 
-	// Fetch current max float for sender
-	maxFloat, err := r.sm.MaxFloat(sender)
-	if err != nil {
-		return nil, err
-	}
+	var maxFloat *big.Int
+	if !r.cfg.IgnoreSenderReserveRequirements {
+		// Fetch current max float for sender
+		var err error
+		maxFloat, err = r.sm.MaxFloat(sender)
+		if err != nil {
+			return nil, err
+		}
 
-	if faceValue.Cmp(maxFloat) > 0 {
-		// If faceValue > maxFloat
-		// Set faceValue = maxFloat
-		faceValue = maxFloat
+		if faceValue.Cmp(maxFloat) > 0 {
+			// If faceValue > maxFloat
+			// Set faceValue = maxFloat
+			faceValue = maxFloat
+		}
+	} else {
+		deposit, err := r.sm.SenderDeposit(sender)
+		if err != nil {
+			return nil, err
+		}
+		if deposit.Cmp(faceValue) < 0 {
```

Comment:
Don't you need to take into account the current balance like it's done [here](https://github.com/livepeer/go-livepeer/blob/92c526dc87734a80232fa529c67418df65119855/pm/sendermonitor.go#L225)?